.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Did You find a bug in PMDK? Please let us know.
```

Comment:
if I'm not mistaken in English there's no usage of capital letter in "You"

Reply:
Done.